src/components/scenes/RequestScene.tsx

```diff
@@ -364,7 +364,7 @@ export class RequestSceneComponent extends React.Component<Props, State> {
               />
             )}
           />
-          <TouchableOpacity disabled={addressExplorerDisabled} onPress={this.handleAddressBlockExplorer}>
+          <TouchableOpacity accessible={false} disabled={addressExplorerDisabled} onPress={this.handleAddressBlockExplorer}>
```

Comment:
What children were made accessible here?

Reply:
The EdgeText is made accessible